spec/models/miq_request_task_spec.rb

```diff
@@ -0,0 +1,24 @@
+describe MiqRequestTask do
+  let(:options) { {'a' => 1, 'b' => 2} }
+  let(:update_options) { {'a' => 'one', 'c' => 'three'} }
+  let(:miq_request) do
+    FactoryGirl.build(:miq_host_provision_request,
```

Comment:
@mkanoor Can we use the base `miq_request` class here instead of a subclass?

Reply:
@gmcculloug Using the base miq_request class I get error on this line
https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_request.rb#L19

The TASK_DESCRIPTION is not defined. Even if I have a message attribute defined it seems to kick in the default_value_for